libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -92,6 +92,9 @@ void AP_AHRS_NavEKF::update(void)
     update_SITL();
 #endif
 
+    // update AOA and SSA
```

Comment:
Great, glad we agree. Please wrap it with:
#if APM_BUILD_TYPE(APM_BUILD_ArduPlane)